doc/source/ccs/setting-up-a-case.rst

```diff
@@ -0,0 +1,327 @@
+.. _ccs_setting_up_a_case:
+
+Setting up a Case
+=================
+
+.. contents::
+    :local:
+
+Configuration
+-------------
+After creating a case, some aspects of the configuration are fixed (any variables in ``env_case.xml``). But many can be changed before calling ``case.setup``, which generates the files required to build and run the case. CIME provides ``xmlquery`` to view and ``xmlchange`` to modify the variables used in this process.
+
+.. _ccs_xmlquery:
+
+Querying
+`````````
+CIME provides the ``xmlquery`` command to view the configuration of a case. The configuration is a collection of variables stored in XML files. These variables are grouped together when defined in the model's configuration.
+
+The following will list all of the current configuration.
+
+.. code-block:: bash
+
+    ./xmlquery --listall
+
+To view a specific group, replace ``<group>`` with the desired group name.
+
+.. code-block:: bash
+
+    ./xmlquery --subgroup <group> --listall
```

Comment:
The xmlquery -p option is one that I use frequently and it should probably be mentioned here.  It prints all of the variables matching a given substring, for example
./xmlquery -p STOP will match all xml variables with STOP in the name: 
```
Results in group run_begin_stop_restart
	STOP_DATE: -999
	STOP_N: 20
	STOP_OPTION: ndays

```

Reply:
Updated this section documenting other useful options, including `-p`.